src/name_bazaar/ui/pages/user_offerings_page.cljs

```diff
@@ -52,10 +52,12 @@
 
 (defn user-offerings []
   (let [search-results (subscribe [:offerings/user-offerings])
-        route-params (subscribe [:district0x/route-params])]
+        route-params (subscribe [:district0x/route-params])
+        r-route-params (subscribe [:district0x/resolved-route-params])]
```

Comment:
You don't really need resolved params here. Doesn't matter if it's name or address here. If it's name, we actually want to display name in title, shared url and so on.